pyproject.toml

```diff
@@ -157,7 +157,7 @@ flake8-docstrings = "^1.5.0"
 google-cloud-storage = "^2.4.0"
 azure-storage-blob = "<12.12.0"
 coveralls = "^3.0.1"
-towncrier = "^21.3.0"
+towncrier = "^22.8.0"
```

Comment:
I would double-check if we don't need to update some CLI towncrier commands we use in the release script with this major upgrade.

Reply:
Ran `towncrier build --yes --version 0.0.0` which is in the release script and `towncrier --draft` which is mentioned in the readme and both worked.

Reply:
Perfect, thank you!